src/FakeClient/FakeClientMain.cpp

```diff
@@ -47,6 +47,8 @@ ABSL_FLAG(bool, enable_cgroup_memory, false, "Collect cgroup and process memory
 namespace {
 std::atomic<bool> exit_requested = false;
 
+constexpr uint16_t kStackDumpSize = 65000;
```

Comment:
I would just move this to `main` after `uint16_t samples_per_second` so that it's closer to where it's used.

Reply:
Done.